lib/consensus/isaac_test.go

```diff
@@ -33,7 +33,7 @@ func TestGetSyncInfoNormal(t *testing.T) {
 	is.nodesHeight[valids[0]] = 10
 	is.nodesHeight[valids[1]] = 10
 	is.nodesHeight[valids[2]] = 10
-	is.nodesHeight[invalids[0]] = 3
+	is.nodesHeight[invalids[0]] = 9
```

Comment:
Why the change ?

Reply:
Sorry for confusing it. I'll revert it :)